BayesOpt/BayesOpt.py

```diff
@@ -310,20 +325,31 @@ def _initialize(self):
         """Generate the initial data set (DOE) and construct the surrogate model"""
         if hasattr(self, 'data'):
             self.logger.warn('initialization is already performed!') 
-            return 
+            return
+
+        if hasattr(self, 'warm_data'):
+            self.logger.info('adding warm data to the initial model')
+            self.data = copy.deepcopy(self.warm_data)
```

Comment:
My original idea was to draw the initial design regardless of the warm data. Mainly because the warm data might be only invalid solutions... do you think that it would be better to skip the initial design if there is warm data?

Reply:
I am not sure now... Let's keep what you have now